src/test/java/com/maxmind/db/ReaderTest.java

```diff
@@ -567,6 +567,38 @@ public TestModelSubdivision(
         }
     }
 
+    @Test
+    public void testDecodeWrongTypeSubdivisions() throws IOException {
+        this.testReader = new Reader(getFile("GeoIP2-City-Test.mmdb"));
+        DeserializationException ex = assertThrows(DeserializationException.class,
+        () -> this.testReader.get( InetAddress.getByName("2.125.160.216"),TestWrongModelSubdivisions.class));
```

Comment:
Indenting this would make the code clearer given that it is a parameter for `assertThrows`.